### PR TITLE
Set Pause and honk on controller

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

**Comment by m4rtint** [comment]
[Allow controller to select pause and end game](https://trello.com/c/zWVtKqkg/98-allow-controller-to-select-pause-and-end-game)

**Comment by m4rtint** [comment]
[Allow controller to select pause and end game](https://trello.com/c/zWVtKqkg/98-allow-controller-to-select-pause-and-end-game)

File: Assets/Script/Managers/InputManager.cs [hunk 1]
```diff
@@ -75,39 +75,34 @@ public static bool HandBreak()
         /// </summary>
         public static void PauseAndPlay()
         {
-            if (Input.GetButtonUp("Pause"))
+            if (Input.GetButtonDown("Pause") || IsStartPressed())
```
Comment (Gursaanj) [inline Assets/Script/Managers/InputManager.cs]: I understand that were are using the start button on controllers for the pause button , so should we rename the bool to IsStartButtonPressed() instead? 

Could be a confusing naming convention later on if we give the controllers the ability to start the game from the main menu, and in turn, using a similar naming convention for a bool

Comment (m4rtint) [reply Assets/Script/Managers/InputManager.cs]: IsStartButtonPressed - the prefix Is, usually means that it's returning a boolean.

**Pull request merged**